AllReadyApp/Web-App/AllReady/Areas/Admin/Features/Tasks/AssignTaskCommandHandlerAsync.cs

```diff
@@ -69,7 +69,7 @@ public AssignTaskCommandHandlerAsync(AllReadyContext context, IMediator mediator
                 var taskSignupsToRemove = task.AssignedVolunteers.Where(taskSignup => message.UserIds.All(uid => uid != taskSignup.User.Id)).ToList();
                 taskSignupsToRemove.ForEach(taskSignup => task.AssignedVolunteers.Remove(taskSignup));
 
-                if (campaignEvent.EventType != EventTypes.EventManaged)
+                if (true /*campaignEvent.EventType != EventTypes.EventManaged*/)
```

Comment:
If deleting the EventManaged per @tonysurma I'm sure we can remove the conditional blocks as appropriate (which I think was related to your query on if we should keep them). Should be safe to remove.